contrib/node/src/python/pants/contrib/node/tasks/node_bundle.py

```diff
@@ -0,0 +1,61 @@
+# coding=utf-8
+# Copyright 2015 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+from __future__ import (absolute_import, division, generators, nested_scopes, print_function,
+                        unicode_literals, with_statement)
+
+import os
+
+from pants.base.build_environment import get_buildroot
+from pants.fs import archive
+
+from pants.contrib.node.tasks.node_paths import NodePaths
+from pants.contrib.node.tasks.node_task import NodeTask
+
+
+class NodeBundle(NodeTask):
+  """Create an archive bundle of NodeModule targets."""
+
+  @classmethod
+  def register_options(cls, register):
+    super(NodeBundle, cls).register_options(register)
+    # Some node modules depend on symlink to function.  Thus we can only support archival type
+    # that can preserve symlinks.
+    register('--archive', choices=list(archive.TYPE_NAMES_PRESERVE_SYMLINKS),
```

Comment:
In the JVM case, we deprecated Task level options and moved them to the targets themselves, because it helped to make the build more reproducible (ie, you don't need to tell someone to use certain command line flags to build your target).

In this case, doing that would require a `node_bundle(dependencies=[':x', ':y'], archive='tar')` target or something, but that's probably a good idea anyway.